scripts/release.js

```diff
@@ -1,16 +1,25 @@
 require('./import-expansions');
 
 const path = require('path');
-const { release } = require(path.join('..', 'packages', 'build'));
+const { release, BuildVariant } = require(path.join('..', 'packages', 'build'));
 const config = require(path.join(__dirname, '..', 'config', 'build.conf.js'));
 
 /**
  * Run the release process.
  */
 const runRelease = async() => {
+  if (process.argv.includes('--dry')) config.dryRun = true;
```

Comment:
we normally always use `{` `}` for blocks with ifs, weird that eslint didn't complain about that

Reply:
Sure, done :)